SPEC.md

```diff
@@ -88,6 +88,14 @@ The response of the operation may be delivered synchronously (inline), or asynch
 - `callback`: Optional. If the operation is asynchronous, the handler should invoke this URL once the operation's
   result is available.
 
+#### Request Headers
+
+A client may attach arbitrary headers to the request.
+
+Headers that start with the `Nexus-Callback-` prefix are expected to be attached to the callback request when invoked by
+the handler. The callback request should strip away the `Nexus-Callback-` prefix. E.g if a Start Operation request
```

Comment:
nit: This is a must, not a should, right? I think we could say that the handler must strip the prefix before delivering the callback to the client.

Reply:
Yes, thanks!